### DISCUSSION TITLE
[Docs] Incorrect link in SQL Databases Tutorial (Alembic Note )

### DISCUSSION THREAD

**Discussion by micallefdaniel** [fastapi/fastapi#10944] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
You can find an example of Alembic in a FastAPI project in the templates from [Project Generation - Template](../project-generation.md){.internal-link target=_blank}. Specifically in <a href="https://github.com/tiangolo/full-stack-fastapi-postgresql/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/alembic/" class="external-link" target="_blank">the `alembic` directory in the source code</a>.
```

### Description

Link to Alembic example in [SQL Databases Tutorial](https://fastapi.tiangolo.com/tutorial/sql-databases/#alembic-note) is broken after merging [Discard Cookiecutter PR](https://github.com/tiangolo/full-stack-fastapi-postgresql/pull/553).

```
<a href="https://github.com/tiangolo/full-stack-fastapi-postgresql/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/alembic/" class="external-link" target="_blank">the `alembic` directory in the source code</a>.
```

Should be changed to:
```
<a href="https://github.com/tiangolo/full-stack-fastapi-postgresql/tree/master/src/backend/app/alembic" class="external-link" target="_blank">the `alembic` directory in the source code</a>.
```

### Operating System

Linux, Windows, macOS

### Operating System Details

N/A

### FastAPI Version

0.109.0

### Pydantic Version

2

### Python Version

3.10

### Additional Context

N/A

**Comment by Kludex** [comment, marked as answer]
- Create https://github.com/tiangolo/fastapi/issues/10945 from this discussion. Thanks.

**Reply by alejsdev** [reply]
This was solved in https://github.com/tiangolo/fastapi/pull/10716 🪄
